public/app/core/components/AppChrome/TopBar/SingleTopBar.tsx

```diff
@@ -98,11 +104,40 @@ export const SingleTopBar = memo(function SingleTopBar({
           <TopSearchBarCommandPaletteTrigger />
           {unifiedHistoryEnabled && !isSmallScreen && <HistoryContainer />}
           {!isSmallScreen && <QuickAdd />}
-          {enrichedHelpNode && (
-            <Dropdown overlay={() => <TopNavBarMenu node={enrichedHelpNode} />} placement="bottom-end">
-              <ToolbarButton iconOnly icon="question-circle" aria-label={t('navigation.help.aria-label', 'Help')} />
-            </Dropdown>
-          )}
+          {enrichedHelpNode &&
+            (enrichedHelpNode.hideFromTabs && availableComponents.has('grafana-grafanadocsplugin-app') ? (
+              (() => {
+                const componentId = getComponentIdFromComponentMeta('grafana-grafanadocsplugin-app', {
```

Comment:
is availableComponents populated at first render? the plugins load async right, so will this first render as the normal dropdown and then later switch after the docs plugin has loaded?